ZipBuilder/Sources/FirebaseReleaser/Tags.swift

```diff
@@ -36,12 +36,43 @@ enum Tags {
         fatalError("Non-Firebase pod \(pod.name) is missing a version")
       }
       let tag = pod.name.replacingOccurrences(of: "Google", with: "") + "-" + version
-      createTag(gitRoot: gitRoot, tag: tag)
+      createTag(gitRoot: gitRoot, tag: tag, afterDelete: afterDelete)
     }
   }
 
-  private static func createTag(gitRoot: URL, tag: String) {
+  static func updateTags(gitRoot: URL) {
+    create(gitRoot: gitRoot, afterDelete: true)
+  }
+
+  private static func createTag(gitRoot: URL, tag: String, afterDelete: Bool) {
+    if afterDelete {
+      verifyDeleteIsOK()
+      Shell.executeCommand("git tag --delete \(tag)", workingDir: gitRoot)
+      Shell.executeCommand("git push --delete origin \(tag)", workingDir: gitRoot)
+    }
     Shell.executeCommand("git tag \(tag)", workingDir: gitRoot)
     Shell.executeCommand("git push origin \(tag)", workingDir: gitRoot)
   }
+
+  private static func verifyDeleteIsOK() {
```

Comment:
optional: I could not realize what the method does until I read the implementation and usage. A name like `verifyTagsAreSafeToDelete` looks a bit more informative to me.

Reply:
Done